### PR TITLE
Integrate Bandsintown Amplified ad network

### PR DESCRIPTION
**Pull request by ZachNusbaum**
We are integrating the Bandsintown Amplified ad network with the `<amp-ad>` tag.  This will allow our publishers to serve our banner creatives properly on their AMP-powered pages.

Intent to implement: Closes ampproject/amphtml#34038

cc ampproject/wg-monetization

### CODE REVIEW

**Comment by ZachNusbaum** [comment]
Hi @kristoferbaxter!

Hope you're having a great start to the week!

This is my first time developing for AMP so I wanted to check in with you about a timeline. We have a client who is checking in with us constantly for updates and it would be great to be able to set expectations with them.  Could you let me know how long the approval process typically takes?

Thanks in advance!
Zach

**Comment by kristoferbaxter** [comment]
Apologies for missing this PR, I've assigned both @powerivq and @calebcordry to take a look since it's a monetization based change.

**Comment by ZachNusbaum** [comment]
Hi @calebcordry and @powerivq - Happy Thursday!

I was wondering if you have an estimate as far as the timeline for this to be approved?  We'd like to be able to update our stakeholders if possible.

Best,
Zach

File: ads/ads.extern.js [hunk 1]
```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+import {data} from 'autoprefixer';
```
Comment (calebcordry) [inline ads/ads.extern.js:17]: nit: please delete

File: ads/vendors/amplified.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
```
Comment (calebcordry) [inline ads/vendors/amplified.js:2]: nit: 2021

**Review by calebcordry — CHANGES_REQUESTED** [review]
Again sorry for delay. Just a couple nits.

**Comment by ZachNusbaum** [comment]
> Again sorry for delay. Just a couple nits.

@calebcordry No worries!  I've made the suggested changes and resolved any merge conflicts.

Best,
Zach

**Comment by calebcordry** [comment]
Looks like just a few lint errors left. You should be able to see them if you follow the CircleCI link above, or you can run `amp lint --local_changes` locally.

**Comment by ZachNusbaum** [comment]
@calebcordry The lint errors have been fixed

Thanks!

File: 3p/integration.js [hunk 3]
```diff
@@ -78,6 +78,7 @@ import {adyoulike} from '#ads/vendors/adyoulike';
 import {affiliateb} from '#ads/vendors/affiliateb';
 import {aja} from '#ads/vendors/aja';
 import {amoad} from '#ads/vendors/amoad';
+import {amplified} from '#ads/vendors/amplified';
```
Comment (calebcordry) [inline 3p/integration.js:81]: I think we no longer need changes in this file. This likely changed when this PR was out for review. @powerivq to confirm.

Comment (powerivq) [reply 3p/integration.js:81]: Yes. You are correct.

**Review by powerivq — CHANGES_REQUESTED** [review]
All good except let's revert 3p/integration.js changes

**Comment by ZachNusbaum** [comment]
@powerivq reverted the changes to 3p/integration.js

**Review by calebcordry — APPROVED** [review]

**Review by powerivq — APPROVED** [review]

**Comment by ZachNusbaum** [comment]
@calebcordry Thank you for approving!
Will this be available for our publishers to start using on their AMP pages immediately, or is there a release timeline?

**Comment by calebcordry** [comment]
@ZachNusbaum you're welcome! This should already be available in nightly if you want to test it there. It should hit beta this week, and prod next week. For more detail see https://amp.dev/documentation/guides-and-tutorials/learn/spec/release-schedule/

**Pull request merged**